galasa-extensions-parent/dev.galasa.cps.etcd/src/main/java/dev/galasa/cps/etcd/internal/Etcd3ConfigurationPropertyStore.java

```diff
@@ -118,15 +118,15 @@ public void setProperty(@NotNull String key, @NotNull String value) throws Confi
 
     @Override
     public Map<String, String> getPropertiesFromNamespace(String namespace) {
-        ByteSequence empty = ByteSequence.from("\0", UTF_8);
+        ByteSequence bsNamespace = ByteSequence.from(namespace, UTF_8);
```

Comment:
you need to tack a dot at the end.

there will be a cics namespace and cicsteam namespace.

so asking for cics should be translated to cics.

Reply:
Done. 👍